setup.py

```
@@ -0,0 +1,50 @@
+#!/usr/bin/env python
```

Ruff (0.13.3)

Shebang is present but file is not executable

(EXE001)